kratos/utilities/variable_utils.h

```diff
@@ -753,6 +753,25 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
         });
     }
 
+    /**
+     * @brief Set the Historical Variables To Zero
+     * This method sets the provided list of variables to zero in the nodal historical database
+     * @tparam TVariableArgs Variadic template argument representing the variables types
+     * @param rNodes Nodes container to be initialized to zero
+     * @param Step Step of the solution step data which needs to be set to zero
+     * @param rVariableArgs Variables to be set to zero
+     */
+    template<class... TVariableArgs>
+    static void SetHistoricalVariablesToZero(
+        NodesContainerType& rNodes,
+        const int Step,
+        const TVariableArgs&... rVariableArgs)
+    {
+        block_for_each(rNodes, [&](auto& rNode){(
+            AuxiliaryHistoricalValueSetter<typename TVariableArgs::Type>(rVariableArgs, rVariableArgs.Zero(), rNode, Step), ...);
+        });
+    }
+
```

Comment:
How about being a bit more general and make it something like `SetHistoricalVariablesTo`, and then provide the value to set them to?